### ISSUE TITLE
Node crashed: "fatal runtime error: failed to initiate panic, error 5"

### ISSUE THREAD

**Issue by dorin100** [input-output-hk/jormungandr#924]
**Describe the bug**
Issue from the community - https://github.com/input-output-hk/shelley-testnet/issues/105 
The node is crashing with `fatal runtime error: failed to initiate panic, error 5`

**Mandatory Information**
```
jcli 0.5.6 (HEAD-7ab929e+, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
jormungandr 0.5.6 (HEAD-7ab929e+, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
```

**Additional context**

**Operating System**:
- NAME="Ubuntu"
- VERSION="18.04.3 LTS (Bionic Beaver)"
- ID=ubuntu
- ID_LIKE=debian
- PRETTY_NAME="Ubuntu 18.04.3 LTS"
- VERSION_ID="18.04"

- Node Config
```
log:
format: "plain"
level: "info"
output: "stderr"
p2p:
listen_address: "/ip4/0.0.0.0/tcp/3100"
public_address: "/ip4/{my_public_ip_here}/tcp/3100"
trusted_peers:
- "/ip4/3.123.177.192/tcp/3000"
- "/ip4/3.123.155.47/tcp/3000"
- "/ip4/52.57.157.167/tcp/3000"
- "/ip4/3.112.185.217/tcp/3000"
- "/ip4/18.140.134.230/tcp/3000"
- "/ip4/18.139.40.4/tcp/3000"
- "/ip4/3.115.57.216/tcp/3000"
topics_of_interest:
messages: "high"
blocks: "high"
rest:
listen: "127.0.0.1:3101"
storage: "/home/shelley/testnet/storage"
```
- Node logs
[failedtoinitiate.panicerror.5nohup.out.txt](https://github.com/input-output-hk/jormungandr/files/3717681/failedtoinitiate.panicerror.5nohup.out.txt)
```
Oct 09 04:12:04.474 INFO client P2P connection closed, node_id: 7639909459871603480, peer_addr: 165.227.201.249:3000, task: network
thread '' panicked at 'failed to allocate an alternative stack', src/libstd/sys/unix/stack_overflow.rs:134:13
note: run with RUST_BACKTRACE=1 environment variable to display a backtrace.
thread '' panicked at 'failed to allocate an alternative stack', src/libstd/sys/unix/stack_overflow.rs:134:13
thread '' panicked at 'failed to allocate an alternative stack', src/libstd/sys/unix/stack_overflow.rs:134:13
thread '' panicked at 'failed to allocate an alternative stack', src/libstd/sys/unix/stack_overflow.rs:134:13
fatal runtime error: failed to initiate panic, error 5
```

**Comment by mzabaluev** [comment]
Hmm, we've got a stack overflow somewhere.

**Comment by NicolasDP** [comment]
I think this is now fixed. It seems that could have been an issue fixed as part of `0.7.0-rc3`.

@dorin100 , please reopen if you still see this issue
